pkg/frontend/adminactions/delete_managedresource.go

```diff
@@ -0,0 +1,57 @@
+package adminactions
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"regexp"
+	"strings"
+
+	"github.com/Azure/go-autorest/autorest/azure"
+
+	"github.com/Azure/ARO-RP/pkg/util/azureclient"
+	"github.com/Azure/ARO-RP/pkg/util/loadbalancer"
+)
+
+var frontendIPConfigurationPattern = `(?i)^/subscriptions/(.+)/resourceGroups/(.+)/providers/Microsoft\.Network/loadBalancers/(.+)/frontendIPConfigurations/([^/]+)$`
```

Comment:
This is an improvement IMO, thanks for making the change!
I agree with others that we should never delete private endpoints and storage accounts - if possible, before merge we should add a check before line 38 that bails out if one of these resources is attempted to be deleted.